src/dateUtils.ts

```diff
@@ -38,56 +40,80 @@ export const getStartDay = (firstDayOfMonth: Date) => {
   return startDay;
 };
 
-
 export const getFormattedDate = (date: Date) => {
   return date.toLocaleDateString('en-US', {
     year: 'numeric',
     month: 'short',
   });
 };
 
-
-export const isDateFromThisMonth = (currentDate: Date, cellDate:Date): boolean => {
+export const isDateFromThisMonth = (
+  currentDate: Date,
+  cellDate: Date
+): boolean => {
   return (
     cellDate.getFullYear() === currentDate.getFullYear() &&
     cellDate.getMonth() === currentDate.getMonth()
   );
 };
 
-export const getFirstDayOfMonth = (date:Date) => {
-  return new Date(
-    date.getFullYear(),
-    date.getMonth(),
-    1
-  );
-}
+export const getFirstDayOfMonth = (date: Date) => {
+  return new Date(date.getFullYear(), date.getMonth(), 1);
+};
 
-export const getLastMonthsFirstDayInMonthCalendar = (firstDayOfCurrentMonth:Date) => {
+export const getLastMonthsFirstDayInMonthCalendar = (
+  firstDayOfCurrentMonth: Date
+) => {
   return new Date(
     firstDayOfCurrentMonth.getFullYear(),
     firstDayOfCurrentMonth.getMonth(),
     firstDayOfCurrentMonth.getDate() - getStartDay(firstDayOfCurrentMonth) + 1
   );
-}
+};
 
-export const getCellDate = (baseDate:Date, offset:number) => {
+export const getCellDate = (baseDate: Date, offset: number) => {
   const cellDate = new Date(baseDate);
   cellDate.setDate(cellDate.getDate() + offset);
   return cellDate;
-}
+};
 
 export const getPreviousMonth = (displayedMonthDate: Date) => {
   return new Date(
     displayedMonthDate.getFullYear(),
     displayedMonthDate.getMonth() - 1,
     1
-  )
-}
+  );
+};
 
 export const getNextMonth = (displayedMonthDate: Date) => {
   return new Date(
     displayedMonthDate.getFullYear(),
     displayedMonthDate.getMonth() + 1,
     1
-  )
-}
\ No newline at end of file
+  );
+};
+
+export const addOneHour = (hour: number): string => {
+  const addition = hour + 1;
+  return formatHourMinutesForInputForm(addition);
+};
+
+export const formatHourMinutesForInputForm = (hour: number): string => {
```

Comment:
```suggestion
export const formatTimeForModalForm = (hour: number): string => {
```
smth like that maybe